synapse/lib/stormlib/edgeinfo.py

```diff
@@ -0,0 +1,145 @@
+import datetime
+
+import synapse.exc as s_exc
+import synapse.common as s_common
+
+import synapse.lib.stormtypes as s_stormtypes
+
+edge_set_desc = '''
+Set a documentation string for an edge that has been added to the graph.
+
+Example:
+    edge.set seen "description for the seen edge verb"
+'''
+
+edge_get_desc = '''
+Get the documentation string for an edge verb.
+'''
+
+edge_del_desc = '''
+Delete the documentation entry for a given edge verb.
+
+Note: If the edge is re-added to the graph the entry will re-populate.
+'''
+
+edge_list_desc = '''
+List all edges that have been added to the graph with their associated documentation.
+'''
+
+stormcmds = [
+    {
+        'name': 'edge.set',
```

Comment:
either `edge.set <edge> <prop> <valu>` or `edge.doc.set`

Reply:
( also, probably prefix with `model.` to make consistent with eventual other model commands )